src/style.css

```diff
@@ -1,3 +1,78 @@
+:root {
+  --link-default: black;
+  --link-active: #00acdc;
+}
+
+html {
+  font-family: "Roboto", monospace;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+img {
+  height: 100%;
+  width: 100%;
+}
+
+nav {
```

Comment:
Don't forget to check the checklist.
![image](https://github.com/mate-academy/layout_moyo-header/assets/54860641/d39d1ca5-453f-473c-a6ea-9d41d1efefc8)